content/20.pharmaceuticals.md

```diff
@@ -25,13 +27,14 @@ Previous emergent viral threats had a much lower death toll that was not felt on
 For example, SARS-CoV-1 was controlled within 9 months of its appearance, whereas SARS-CoV-2 remains a significant global threat almost 1.5 years later.
 ]({{csse_deaths_figure}} "Global COVID-19 deaths"){#fig:csse-deaths secno=1}
 
-SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat closely parallels the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1).
+SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat initially closely paralleled the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1).
 The first documented case of COVID-19 was reported in Wuhan, China in November 2019, and the disease quickly spread worldwide during the early months of 2020.
-The first case of SARS was reported in November 2002 in the Guangdong Province of China, and it spread within China and then into several countries across continents during the first half of 2003 [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81; @doi:10.3390/ijerph17113973].
+In comparison, the first case of SARS was reported in November 2002 in the Guangdong Province of China, and it spread within China and then into several countries across continents during the first half of 2003 [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81; @doi:10.3390/ijerph17113973].
 In fact, genome sequencing quickly revealed the virus causing COVID-19 to be a novel betacoronavirus closely related to SARS-CoV-1 [@doi:10.46234/ccdcw2020.017].
 While similarities between these two viruses are unsurprising given their close phylogenetic relationship, there are also some differences in how the viruses affect humans.
-SARS-CoV-1 infection is severe, with an estimated death rate for SARS of 9.5% [@doi:10.1093/ajcp/aqaa029], while estimates of the death rate associated with COVID-19 are much lower, at approximately 2% [@individual-pathogenesis].
+SARS-CoV-1 infection is severe, with an estimated case fatality rate (CFR) for SARS of 9.5% [@doi:10.1093/ajcp/aqaa029], while estimates of the CFR associated with COVID-19 are much lower, at approximately 2% [@individual-pathogenesis].
 SARS-CoV-1 is highly contagious via droplet transmission and has a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
+<!--We should say at minimum that there's debate over droplets vs aerosols, as seen for instance in this https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(21)00869-2/fulltext?hss_channel=tw-27013292 and the resulting controversy-->
```

Comment:
I agree with adding this line.

Reply:
Sorry I missed this until today! CFR = case fatality ratio, IFR = infection fatality ratio. "Case" means person diagnosed with COVID-19, "infection" means person infected with SARS-CoV-2. For most infectious diseases, CFR > IFR, because some people are asymptomatic. Also, as we do more testing, CFR generally decreases, as more people are diagnosed.